scss/atoms/typography/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
As seen in the [HTML style guide](http://contribute.jquery.org/style-guide/html/#doctype) please lowercase this.